Tests/Packet++Test/Tests/TcpTests.cpp

```diff
@@ -321,7 +321,7 @@ PTF_TEST_CASE(TcpPacketCreation2)
 	pcpp::TcpOption qsOption = tcpLayer.addTcpOptionAfter(pcpp::TcpOptionBuilder(pcpp::TCPOPT_QS, nullptr, PCPP_TCPOLEN_QS), pcpp::TCPOPT_MSS);
 	PTF_ASSERT_TRUE(qsOption.isNotNull());
 	PTF_ASSERT_TRUE(qsOption.setValue(htobe32(9999)));
-	PTF_ASSERT_TRUE(tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionEnumType::Snack, htobe32(1000))).isNotNull());
+	PTF_ASSERT_TRUE(tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionEnumType::Snack, (uint32_t)htobe32(1000))).isNotNull());
```

Comment:
please use `reinterpret_cast`

Reply:
Seems the macro is not working correctly and is returning initializer_list according to MSVC. 

I updated the cast

Reply:
It turns out that reinterpret_cast is not allowed by MSVC either, error msg: ` error C2440: 'reinterpret_cast': cannot convert from 'unsigned long' to 'uint32_t'`. 

I update it to `static_cast`